src/refresh.js

```diff
@@ -0,0 +1,35 @@
+const getLocalStorage = () => {
+  const getObject = 'SeEHnRhRvxVjE8M8I44H';
+  const urlIs = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${getObject}/scores/`;
+  fetch(urlIs, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json()).then((json) => {
+    let count = 0;
+    const getJson = json.result;
+    getJson.forEach((line) => {
+      const game = {
+        name: line.user,
+        score: line.score,
+      };
+      const buildLine = `
+        ${game.name}: ${game.score}
+      `;
+      const getContainer = document.getElementById('recent');
+      const createDiv = document.createElement('div');
```

Comment:
- [ ] When I refresh my browser, all the scores disappear and when I click on refresh, all the scores get added multiple times. The scores should persist on page refresh and the refresh button should only update the latest score and not all the scores. Kindly fix that 👍🏽 

![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/63473584/186514686-2f44baf4-1884-4328-b583-2c380af65fc5.gif)